mhr_api/src/mhr_api/models/db2/manuhome.py

```diff
@@ -200,6 +227,42 @@ def registration_json(self):
             man_home['notes'] = Db2Manuhome.__sort_notes(notes)
         return man_home
 
+    @classmethod
+    def find_summary_by_mhr_number(cls, mhr_number: str):
+        """Return the MH registration summary informaiton matching the MHR number."""
```

Comment:
Super minor but typo in `information`

Reply:
Nice catch, thanks, description updated in latest commit.